apps/condo/domains/property/schema/Property.js

```diff
@@ -46,17 +54,23 @@ const Property = new GQLListSchema('Property', {
             isRequired: true,
             hooks: {
                 validateInput: async ({ resolvedData, fieldPath, addFieldValidationError, context, existingItem, operation }) => {
-                    const value = resolvedData[fieldPath]
```

Comment:
It’s looks like redundant refactoring. And you probably change the behavior

Reply:
restored prev. version